OpenTissue/gpu/image/io/image_read.h

```diff
@@ -1,91 +1,180 @@
-#ifndef OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
-#define OPENTISSUE_GPU_IMAGE_IO_IMAGE_READ_H
 //
 // OpenTissue Template Library
 // - A generic toolbox for physics-based modeling and simulation.
 // Copyright (C) 2008 Department of Computer Science, University of Copenhagen.
 //
 // OTTL is licensed under zlib: http://opensource.org/licenses/zlib-license.php
 //
-#include <OpenTissue/configuration.h>
+#pragma once
 
-#include <OpenTissue/gpu/image/image.h>
-#include <OpenTissue/gpu/image/io/image_il_wrap.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <png.h>
 
 #include <iostream>
 #include <string>
+#include <vector>
+
+#include <OpenTissue/configuration.h>
+#include <OpenTissue/gpu/image/image.h>
 
-namespace OpenTissue
+namespace OpenTissue {
+namespace image {
+/**
+* Read Image.
+*
+* @param   filename
+* @param   image
+* @return               If succesful then the return value is true
+*                       otherwise it is false.
+*/
+bool read(std::string const & filename,
```

Comment:
OT have used the convention of writing commas a little differently on the next line

Reply:
Got it, will revert.